src/database.ts

```diff
@@ -344,7 +344,7 @@ class Database extends common.GrpcServiceObject {
   databaseDialect?: EnumKey<
     typeof databaseAdmin.spanner.admin.database.v1.DatabaseDialect
   > | null;
-  observabilityConfig: ObservabilityOptions | undefined;
+  observabilityOptions_?: ObservabilityOptions;
```

Comment:
Can you try using "private" keyword . Also it should be `_observabilityOptions` right ?

Reply:
I was following the precedent in this library of `formattedName_` as I hadn't seen other variables beginning with the underscore but sure I can update that. The only problem about declaring them with the keyword private is that between classes we actually need to read and set the value and adding a setter seems like overkill. What I could do is set it to `readonly` and infer it only from the constructor as well as having set it as _observabilityOptions as you suggested.

Reply:
So @surbhigarg92 I don't believe that reasonable customers expect to set `formattedName_` nor would a reasonable customer change the now `_observabilityOptions`, even more, making a setter or getter is unnecessary exposure.